src/collection/cereal.rs

```diff
@@ -110,6 +112,65 @@ impl<'de> Deserialize<'de> for Template {
     }
 }
 
+impl<'de> Deserialize<'de> for QueryParameters {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        struct QueryParametersVisitor;
+
+        impl<'de> Visitor<'de> for QueryParametersVisitor {
+            type Value = QueryParameters;
+
+            fn expecting(
+                &self,
+                formatter: &mut std::fmt::Formatter,
+            ) -> std::fmt::Result {
+                formatter.write_str("a sequence or a map of key-value pairs")
+            }
+
+            fn visit_seq<A>(self, mut seq: A) -> Result<Self::Value, A::Error>
+            where
+                A: SeqAccess<'de>,
+            {
+                let mut query: Vec<(String, Template)> = vec![];
+                while let Some(value) = seq.next_element::<String>()? {
+                    match value.split_once('=') {
+                        Some((key, value)) => {
+                            query.push((
+                                key.to_string(),
+                                Template::try_from(value.to_string()).unwrap(),
+                            ));
+                        }
+                        None => {
+                            query.push((
+                                value.to_string(),
+                                Template::try_from("".to_string()).unwrap(),
+                            ));
+                        }
```

Comment:
I think we should raise an error here instead. If the user just put a param name with no `=` it was probably a mistake. If they meant to include the param with no value, they could put `key=`